### PR TITLE
updated desktop launcher

### PR DESCRIPTION
**Pull request by jspauldo98**
This change is [Reviewable](https://reviewable.io/reviews/boomboxinc/salt/29)

### CODE REVIEW

File: desktop/src/com/spaulding/salt/desktop/DesktopLauncher.java [hunk 1]
```diff
@@ -7,6 +7,10 @@
 public class DesktopLauncher {
 	public static void main (String[] arg) {
 		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
+		config.title = "Ladder";
```
Comment (ConnorGBrewster) [inline desktop/src/com/spaulding/salt/desktop/DesktopLauncher.java]: CHANGE THIS TO SALT

**Comment by ConnorGBrewster** [comment]
@homu-salt r+

**Comment by homu-salt** [comment]
:pushpin: Commit ea3266a has been approved by `ConnorGBrewster`

**Comment by homu-salt** [comment]
:hourglass: Testing commit ea3266a with merge d364f4f...

**Comment by homu-salt** [comment]
:sunny: Test successful - [travis](https://travis-ci.org/boomboxInc/Salt/builds/130689902)

**Pull request merged**